diff --git a/src/app/kde/CMakeLists.txt b/src/app/kde/CMakeLists.txt
index dcac9cee..72d24182 100644
--- a/src/app/kde/CMakeLists.txt
+++ b/src/app/kde/CMakeLists.txt
@@ -11,13 +11,7 @@ if(NOT MSVC)
   target_link_libraries(kid3 -lstdc++)
 endif()
 
-install(TARGETS kid3 DESTINATION ${BIN_INSTALL_DIR})
+install(TARGETS kid3 DESTINATION ${KDE_INSTALL_BINDIR})
 install(PROGRAMS ../org.kde.kid3.desktop DESTINATION ${KDE_INSTALL_APPDIR})
 install(FILES ../org.kde.kid3.appdata.xml DESTINATION "${WITH_DATAROOTDIR}/metainfo")
-if(QT_VERSION_MAJOR EQUAL 6)
-  # TODO KF6: If there something defined?
-  if(NOT KXMLGUI_INSTALL_DIR)
-    set(KXMLGUI_INSTALL_DIR "${WITH_DATAROOTDIR}/kxmlgui6")
-  endif()
-endif()
-install(FILES kid3ui.rc DESTINATION ${KXMLGUI_INSTALL_DIR}/kid3)
+install(FILES kid3ui.rc DESTINATION ${KDE_INSTALL_KXMLGUIDIR}/kid3)
